### DISCUSSION TITLE
set proxy server, set timeout, still some links timeout and cannot end

### DISCUSSION THREAD

**Discussion by mu0gua** [projectdiscovery/httpx#281] [General]
set proxy server, set timeout, still some links timeout and cannot end

> command: cat ip.txt | httpx.exe -title -status-code -o r.txt

![image](https://user-images.githubusercontent.com/48984011/118785520-81c16200-b8c3-11eb-822f-e58d46a0a063.png)

**Comment by mu0gua** [comment]
set thread 50 , But the number of connections will exceed 50 after a period of time

**Comment by ehsandeep** [comment]
@mu0gua Can you share an example for this?

> still some links timeout and cannot end
